docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -528,6 +528,153 @@ Required ECS fields when not using {beats}:::
 
 // end::siem-auditbeat-auth-jobs[]
 
+[discrete]
+[[security-authentication]]
+== Security: Authentication
+
+Detect anomalous activity in your ECS-compatible authentication logs.
+
+These configurations are only available if data exists that matches the 
+recognizer query specified in the
+https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/manifest.json#L8[manifest file].
+
+If you create these jobs in the *{ml-app}* app, you must first create the
+recommended
+https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/manifest.json#L7[{data-source}].
+
+// tag::security-authentication-jobs[]
+auth_high_count_logon_events::
+Looks for an unusually large spike in successful authentication events. This can
+be due to password spraying, user enumeration or brute force activity.
+
+Job details:::
+
+* Detects anomalies where the number of events is unusually high and ignores
+cases where the count is zero (using the
+{ml-docs}/ml-count-functions.html#ml-nonzero-count[`high_non_zero_count` function]).
+
+Required {beats} or {agent} integrations:::
+
+* {elastic-endpoint} integration
+
+Required ECS fields:::
+
+* `event.category`
+* `event.outcome`
+
+auth_high_count_logon_events_for_a_source_ip::
+Looks for an unusually large spike in successful authentication events events
```

Comment:
Duplicate `events` here

Reply:
Thanks!  I've created https://github.com/elastic/kibana/pull/112135 to fix that in the module too.